packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2172,6 +2172,17 @@ export class StripeHelper {
       throw error;
     }
 
+    // We make an exception here for customers because we need to get the
+    // subscriptions for the customer.  The Stripe API stopped including
+    // subscriptions for a customer in version 2020-08-27; this ensures
+    // backwards compatibility for our code that's relying on that behavior.
+    if (resourceType === CUSTOMER_RESOURCE) {
+      // @ts-ignore
```

Comment:
Sorry, should've mentioned this before, but why the ts-ignore? Since its a customer here, might as well just do the customer retrieve the normal way so the type-checker can help out?

Reply:
The same reason for the @ts-ignore on line 2186, which is the generic type param `T` not matching the actual return type.